tensorflow/c/experimental/gradients/unified_api_testutil.h

```diff
@@ -56,6 +57,13 @@ Status TestScalarTensorHandle(AbstractContext* ctx, float value,
 Status TestTensorHandleWithDimsFloat(AbstractContext* ctx, float* data,
                                      int64* dims, int num_dims,
                                      AbstractTensorHandle** tensor);
+
+// Get a TensorHandle with given int values and dimensions
+Status TestTensorHandleWithDimsInt(AbstractContext* ctx, int* data, int64* dims,
```

Comment:
Could we use `int64_t* dims` here ? Since some C API functions return `int64_t`, a `int64_t*` is better in my opinion.

Reply:
Yep sounds good.